config.yml

```diff
@@ -15,7 +15,7 @@ APP:
   JOBS_USERNAME: ${JOBS_USERNAME}
   JOBS_PASSWORD: ${JOBS_PASSWORD}
 ROCK:
-  API_URL: ${ROCK_API}
+  API_URL: https://alpha-rock.newspring.cc/api
```

Comment:
Was this just a test thing? Not sure removing the ability to change the version via an environment variable is the right thing to do here.

Reply:
oh yea my bad